### PR TITLE
[RV64_DYNAREC] Added 0F 2E,2F opcode & some fixes in printer

### PR DESCRIPTION
**Pull request by ksco**
Please review this one carefully, don't know if I understand the corresponding SSE infra correctly. I believe it's right, but still not so sure.

### CODE REVIEW

**Comment by ksco** [comment]
test15 failed, investigating.

**Comment by ksco** [comment]
It's actually test16 is failing, but why CI says test15?

**Comment by ptitSeb** [comment]
> test15 failed, investigating.

Because 1 test is skipped and it messed with the count. It's anoying, I know.

**Comment by ksco** [comment]
Side note: cosim doesn't seem to compare flags?

**Comment by ptitSeb** [comment]
> Side note: cosim doesn't seem to compare flags?

No, that's not possible because flags are optimized by the Dynarec. You can only observe consequences of flags (like a conditionnal jump is not correct, or a setxx, things like that...)

**Comment by ksco** [comment]
Can we disable flag optimizations and just calculate all flags when running in `BOX64_DYNAREC_TEST` mode?

**Comment by ptitSeb** [comment]
> Can we disable flag optimizations and just calculate all flags when running in `BOX64_DYNAREC_TEST` mode?

That's not what we want to do. Flags optimisation is an important part of the Dynarec, so we want to compare the Dynarec as-is, not some "carefull" version of it.

**Comment by ksco** [comment]
Ok, makes sense again, thanks for answering. I have no more questions, this PR is now ready for review, CI passed.

File: src/dynarec/rv64/dynarec_rv64_0f.c [hunk 1]
```diff
@@ -200,7 +200,43 @@ uintptr_t dynarec64_0F(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
...
+                FLTS(x2, gd, v0);
+                BEQ_MARK2(x2, xZR);
+                ORI(xFlags, xFlags, 1<<F_CF);
+                B_NEXT_nocond;
+            }
+            MARK2;
+            // else if gd == v0
+            IFX(X_ZF) {
+                FEQS(x2, gd, v0);
+                BEQ_MARK3(x2, xZR);
```
Comment (ptitSeb) [inline src/dynarec/rv64/dynarec_rv64_0f.c:235]: a branch to NEXT could be used here also...

Comment (ksco) [reply src/dynarec/rv64/dynarec_rv64_0f.c:235]: Yeah, will do it in next PR.

**Pull request merged**